Library/Formula/tcc.rb

```diff
@@ -5,11 +5,17 @@ class Tcc < Formula
   url 'http://download.savannah.gnu.org/releases/tinycc/tcc-0.9.26.tar.bz2'
   sha1 '7110354d3637d0e05f43a006364c897248aed5d0'
 
+  option "with-cross", "Build cross compilers"
+
   def install
+    opts = []
```

Comment:
How about moving the entirety of the arguments from below into a `%W[]` array, and append this onto that?